requirements.txt

```diff
@@ -2,3 +2,4 @@
 dpath==2.0.6
 black
 linkml_runtime
+linkml # Sujay suggested this instead
```

Comment:
Is "linkml" in lieu of linkml_runtime, or in addition to?

Reply:
I think @sujaypatil96 was saying use the more inclusive linkml in lieu of linkml-runtime. I'm not convinced that anything outside of linkml-runtime is really needed though. But I haven't checked.